fractal.js

```diff
@@ -32,9 +32,6 @@ components.engine(
   })
 );
 
-const docs = fractal.docs;
```

Comment:
Removed since the list is the only docs element. We don't need to build docs if we're not displaying this element.